internals/api/uuid_test.go

```diff
@@ -0,0 +1,10 @@
+package api_test
+
+import (
+	"github.com/secrethub/secrethub-go/internals/api/uuid"
+)
+
+// UUID returns the pointer to the given uuid.UUID value.
+func UUID(id uuid.UUID) *uuid.UUID {
```

Comment:
What is the purpose of having this function and does it have its own file?

Reply:
It's for test purposes only, because we create a lot of requests there with fake UUIDs. We could also refactor it to call `uuid.New` itself and rename it to something like `fakeUUID`. Would that make things clearer?

Reply:
Why do we need it in the first place? Why not simply call `uuid.New` in the places where you generate a random UUID for test purposes?

Reply:
Because `uuid.New` is used server-side, were we'd also consume the value and not the pointer, as in (almost?) all cases the id is a required field.